packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -177,12 +185,15 @@ export default defineComponent({
       computedInputAttributes,
       fieldListeners: createFieldListeners(emit),
       reset,
+      focus,
+      blur,
     }
   },
 
+  // we will use this while we have 'withConfigTransport' and problem with 'expose' method in 'setup' func
   methods: {
-    focus () { this.input?.focus() },
-    blur () { this.input?.blur() },
+    _focus () { this.input?.focus() },
```

Comment:
Better use `_focus` in setup and define allow user to use focus method using ref, instead of _focus.

![image](https://user-images.githubusercontent.com/23530004/145323344-f02ecd83-8187-47ea-ad85-92d948f975f3.png)
![image](https://user-images.githubusercontent.com/23530004/145323358-73f60d51-3222-4b54-8ee4-bf8142c17c35.png)

Reply:
Fixed it